src/os/locale.zig

```diff
@@ -108,11 +108,8 @@ fn setLangFromCocoa() void {
     }
 
     // Get our preferred languages and set that to the LANGUAGE
-    // env var in case our language differs from our locale. We only
-    // do this when the app is launched from the desktop because then
-    // we're in an app bundle and we are expected to read from our
-    // Bundle's preferred languages.
-    if (internal_os.launchedFromDesktop()) language: {
```

Comment:
I could be wrong, but I spent some time dwelling on this and I don't think we needed this guard. If we launch from a non-desktop source, one of the prior language setting cases should match first. If they don't, we should probably still try this.